### ISSUE TITLE
NativeMessageingHosts manifest file is not created for brave

### ISSUE THREAD

**Issue by popy2k14** [bitwarden/clients#11750]
### Steps To Reproduce

1. Use mint 22 and install howdy, so it identifies you by face
2. Install latest bitwarden desktop linux appimage (currently 2024.10.1)
3. Install Brave browser with latest extension (currently 2024.10.1)
4. Unlock bitwarden desktop with master password
5. Enable settings -> browser extention function
6. start brave, connect to bitwarden server
7. go to settings and try to activate biometrics (settings -> security -> biometrics)
8. It says it, that first "browser extention function" must be enabled in desktop app, which is

### Expected Result

Brave works like google-chrome

### Actual Result

It says it, that first "browser extention function" must be enabled in desktop app, which is

### Additional Context

Sadly, the manifest is just generated for firefox, google-chromium & edge.
See here: https://github.com/bitwarden/clients/blob/f2be150b70827269020f9a2552c01294aac35258/apps/desktop/src/main/native-messaging.main.ts#L193-L213

So with a little workaround and it's now working as it should (google-chrome not installed):

```
mkdir ~/.config/google-chrome/NativeMessagingHosts
cd ~/.config/BraveSoftware/Brave-Browser/NativeMessagingHosts
ln -s ../../../google-chrome/NativeMessagingHosts/com.8bit.bitwarden.json com.8bit.bitwarden.json
```

Would be nice if more chromium based browsers will get supported by the client.

### Operating System

Linux

### Operating System Version

Mint 22

### Installation method

Direct Download (from bitwarden.com)

### Build Version

2024.10.1

### Issue Tracking Info

- [X] I understand that work is tracked outside of GitHub. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by cbbit** [comment]
Hi there,

I am unable to reproduce this issue; it has been escalated for further investigation. If you have more information that can help us, please add it below. 

Thanks!

**Comment by tim-rohrer** [comment]
Hey @popy2k14 , are you manually downloading the `com.8bit.bitwarden.json` manifest to the `google-chrome/NativeMessagingHosts` folder?

**Comment by popy2k14** [comment]
@tim-rohrer i made a symbolic link as stated in the workaround of the first post.

**Comment by tim-rohrer** [comment]
@popy2k14, yes, I did that. But there is nothing to make a link to on my system.

Since you stated above that "google-chrome not installed," I uninstalled it. The other difference is that  Brave on my system is installed with Snap, so my link looks like this:

`com.8bit.bitwarden.json -> /home/tim/.config/google-chrome/NativeMessagingHosts/com.8bit.bitwarden.json`

where `pwd` is `/home/tim/snap/brave/current/.config/BraveSoftware/Brave-Browser/NativeMessagingHosts`

Maybe I need to try installing Brave with `apt`?

**Comment by popy2k14** [comment]
@tim-rohrer sorry i dont have that system anymore to look at it.
Yeah try a few methods of installing, sadly cant remember.
Maybe i installed chrome, set it up, and uninstalled it.

**Comment by undermark5** [comment]
Anyone else coming here without chrome installed you don't have to install chrome, just make the `~/.config/google-chrome/` directory, enable the browser integration setting (disable it if it's already enabled), and then the manifest files will show up.

**Comment by dani-garcia** [comment]
With #16705 Vivaldi and Brave have been explicitly added to Windows and Linux, so this issue should hopefully be solved on the next release.
